src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Syntax/MarkupEndTagSyntax.cs

```diff
@@ -9,36 +9,44 @@ namespace Microsoft.AspNetCore.Razor.Language.Syntax;
 
 internal partial class MarkupEndTagSyntax
 {
+    private SyntaxNode _children;
```

Comment:
For these fields, the roslyn convention is to name them `_lazyX`. I'd like to see us adopt the same convention in razor.